server/osc_dash.py

```
@@ -133,149 +213,372 @@ def api_backtest(
+        try:
+            _collector_proc.wait(timeout=2.0)
+        except Exception:
+            _collector_proc.kill()
+    _collector_proc = None
+    return {"ok": True, "running": False}
+
+
+@app.post("/api/collector/poll-once")
+def api_collector_poll_once():
+    cmd = [sys.executable, "-m", "scripts.collect_ticks", "--once"]
+    res = subprocess.run(cmd, cwd=str(ROOT), capture_output=True, text=True)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)